.github/workflows/release.yaml

```diff
@@ -0,0 +1,24 @@
+name: Uplaod assets to COS
+on:
+  push:
+    tags:
+      - 'v**'
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+jobs:
+  push_assets:
+   runs-on: ubuntu-latest
+   steps:
+     - name: Checkout
+       uses: actions/checkout@v2
+     - name: Upload to COS
+       uses: zkqiang/tencent-cos-action@v0.1.0
+       with:
+          args: |
+            upload -r ./assets/ /${{ github.ref_name }}/assets/ && upload -r ./scripts/ /${{ github.ref_name }}/scripts/ \
```

Comment:
We can delete `./scripts` directory now, it already replaced with cloud-cli. cc @tokers

Reply:
removed